### ISSUE TITLE
Seeesaw Servo Library Gives Unexpected Behavior on ATTiny817 Seesaw Breakout

### ISSUE THREAD

**Issue by ryancasler** [adafruit/Adafruit_seesawPeripheral#7]
I used the following sketch with an ESP32-S2-TFT Feather and a ATTiny817 Seesaw breakout board:
```
#include "Adafruit_seesaw.h"
#include "seesaw_servo.h"
Adafruit_seesaw ss;
seesaw_Servo myservo(&ss); 
void setup() {
  Serial.begin(115200);
  pinMode(TFT_I2C_POWER, OUTPUT);
  digitalWrite(TFT_I2C_POWER, HIGH);
 
  if(!ss.begin()){
    Serial.println("ERROR!");
    while(1) delay(1);
  }
  Serial.println("Seesaw started");
  ss.pinMode(0,OUTPUT);
  ss.digitalWrite(0, LOW);
  myservo.attach(0); 
}

void loop() {
  myservo.write(0);
  delay(1000);
  myservo.write(90);
  delay(1000);
  myservo.write(180);
  delay(1000);
  myservo.write(90);
  delay(1000);
}
```

That resulted in the following waveforms from the servo pin.
![Seesaw 1](https://user-images.githubusercontent.com/30270489/162125065-e59f9267-fc51-4842-932d-3351acca8f15.jpg)
![Seesaw 2](https://user-images.githubusercontent.com/30270489/162125068-ef376b2f-7295-4d2a-9f3e-d39d3f65503f.jpg)

Instead of the normal waveform I would expect for controlling a servo.  Opening this request per request from Adafruii2 from [the forum.](https://forums.adafruit.com/viewtopic.php?f=19&t=189924)
Let me know if there is anything else you need or if there is something else I should test.

**Comment by caternuson** [comment]
That looks like the expected behavior for how the core implements `analogWrite()` for PWM. Good info here:
https://github.com/SpenceKonde/megaTinyCore#pwm-support
and even better info here on timer specifics:
https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/Ref_Timers.md

For the ATtiny817 @ 20 MHz, the PWM frequency ends up being ~1225Hz:
https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/Ref_Timers.md#pwm-frequencies
which compares well with the 1213Hz shown in the captures above.

There is a separate [Servo library](https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/extras/Ref_Timers.md#servo-library) in the core that may need to get used instead of `tone()` and `analogWrite()`?

**Comment by caternuson** [comment]
Just reporting that this example works:
https://github.com/SpenceKonde/megaTinyCore/blob/master/megaavr/libraries/Servo_megaTinyCore/examples/Sweep/Sweep.ino

Tested with a micro servo attached to pin 9 of attiny817 breakout.

**Comment by caternuson** [comment]
Closing. This is nominally fixed by #14. With that PR, building firmware with the ability to create PWM output for servo control is possible.
